rcl/test/rcl/test_events.cpp

```diff
@@ -269,6 +269,71 @@ wait_for_msgs_and_events(
   return RCL_RET_OK;
 }
 
+// @brief Conditional function for determining when the wait_for_msgs_and_events loop is complete.
```

Comment:
nitpick: please use `///` and no `brief`. Also we use `\` rather than `@`.

Reply:
Fixed.